ceno_zkvm/src/e2e.rs

```diff
@@ -1744,6 +1744,18 @@ fn create_proofs_streaming<
         }
     });
     metrics::gauge!("num_shards").set(proofs.len() as f64);
+
+    // Currently, due to mixed usage with other GPU backends,
+    // we need to trim ceno-gpu's memory pool while still retaining 424MB.
+    // Once the GPU backend is unified, skipping this trim
+    // could improve performance by a few seconds.
+    #[cfg(feature = "gpu")]
+    {
+        use gkr_iop::gpu::gpu_prover::*;
```

Comment:
should we move the deleted span time tracking "wait for GPU to free memory" in here? probably with span wording "[ceno] gpu mem pool trim. TODO remove this once cuda backend unified"

Reply:
[59e02dc](https://github.com/scroll-tech/ceno/pull/1201/commits/59e02dcfe94a8f7fbd884fbcaee9318702126929)